app/domain/errors.rb

```diff
@@ -383,6 +383,17 @@ module AuthnJwt
         code: "CONJ00072E"
       )
 
+      NoSuchFieldInToken = ::Util::TrackableErrorClass.new(
+        msg: "No such field '{0}' in the token",
+        code: "CONJ00073E"
+      )
+
+      NoRelevantIdentityProvider = ::Util::TrackableErrorClass.new(
+        msg: "No relevant identity provider found." \
```

Comment:
IMO the term `identity provider` is confusing here, the use case here that the user didn't send an identity in the url and do not configure the `token-app-property` in the policy.

We should change error message maybe to:
Failed to fetch the application identity.
Please do one of the following options:
1. Send the identity in the URL
2. As you wrote

Reply:
Done